doc/CMakeLists.txt

```diff
@@ -66,8 +66,10 @@ endif()
 find_program(TEXI2PDF texi2pdf
              ${CYGWIN_INSTALL_PATH}/bin /bin /usr/bin /usr/local/bin /sbin)
 mark_as_advanced(TEXI2PDF)
+find_program(PDFLATEX pdflatex
+             ${CYGWIN_INSTALL_PATH}/bin /bin /usr/bin /usr/local/bin /sbin)
```

Comment:
:)

Reply:
Yup. But it's not only pdflatex, it's fonts,  packages, and problems with doxygen versions which generate bad latex... Believe me it's a can of worms I spent almost two days to dissect... and the public travis interface (without debugging) is a pain...